test/utils/setup.ts

```diff
@@ -1,14 +1,50 @@
 /*
     Setup -- start dynamodb instance
  */
+import { spawn } from 'child_process'
+const waitPort = require('wait-port')
 import DynamoDbLocal from 'dynamo-db-local'
 
 const PORT = 4567
 
 module.exports = async () => {
+    let dynamodb
 
-    let dynamodb = DynamoDbLocal.spawn({port: PORT})
-    process.env.DYNAMODB_PID = dynamodb.pid
-    process.env.DYNAMODB_PORT = PORT.toString()
-    // console.log('Spawn DynamoDB', dynamodb.pid)
+    if (String(process.env.DOCKER) === 'true') {
+        const args = [`run`, `-p`, `${PORT}:8000`, `amazon/dynamodb-local`]
```

Comment:
re: https://github.com/sensedeep/dynamodb-onetable/pull/97/files#r689922361

Internally on the docker network, 8000/tcp will be running for the container, but here it can be mapped to whatever port is set as `PORT`, on the host machine.

This means that it works the same way as:

```ts
dynamodb = DynamoDbLocal.spawn({ port: PORT })
```

See my suggestion above :+1: